### ISSUE TITLE
Unable to load GDAL support, no heightmap export possible

### ISSUE THREAD

**Issue by PeteMichaud** [Mindwerks/worldengine#229]
I'm on the latest OSX, and I have GDAL installed from the recommended: http://www.kyngchaos.com/software:frameworks (version 1.11 of GDAL apparently. I understand 2.x versions have a problem with imports too).

In the python shell I run `from osgeo import gdal` and it works fine, importing without an issue. 

When I run `worldengine` with options, I get the full output, but near the top of that output is "Unable to load GDAL support, no heightmap export possible" - I want that heightmap!!

Am I doing something wrong?

**Comment by psi29a** [comment]
Have a look here:
https://github.com/Mindwerks/worldengine/blob/master/worldengine/imex/__init__.py

We try importing just like you do, in case that fails we try just importing gdal directly.

Then we give up. For whatever reason, WE can't find your libs. The question becomes is, your "python shell" is aware of your libraries but is your normal shell or however you are running WE?

**Comment by PeteMichaud** [comment]
Yeah, I don't know -- I also saw that bit of code which is why I tried it in the shell. I'm not really a python dev, so I'm not sure how the loading works. 

I just ran `python` from my terminal, and did the import, successfully. Then I exited `python`, and ran `worldengine` from the same directory I ran `python`--that's when I got the error. 

I'm assuming `python` runs under my user, and that `worldengine` does also, so I would also assume that even if only my user had gdal in its $PATH that both would have worked. 

I don't know how to troubleshoot this more. Is there some way to make sure worldengine has a certain path?
